src/components/Sum.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Sum = ({ a, b }) => (
+  <p>{`Sum of ${a} and ${b} is ${+a + +b}`}</p>
```

Comment:
You can pass proops not as a string, but as a JS expression. Then you will not need to use {+ a + + b}